projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/impala_lineage_plugin.py

```diff
@@ -111,8 +111,19 @@ def _is_query_have_lineage(query_statement: str) -> bool:
             if line.endswith("select 1 -- testing if connection is alive."):
```

Comment:
"testing if connection is alive" - this is not standard comment - it's custom comment made by one user of VDK . It cannot be part of the open source project. 

Quire reminder toward the contributing guide: https://github.com/vmware/versatile-data-kit/wiki/How-to-prepare-a-new-PR#self-evaluate-changefeature-proposal 

Particularly 

> Before suggesting a change/feature, think if this change only serves your needs, or it is broader, that is good for the project itself because it serves multiple users. If it is a very specific change limited in its usefulness, can it be generalised to meet the needs of more users? If yes, propose the more broadly useful feature.

Reply:
That will not catch comments made with "/* comment */" style, right ? 

Which might be ok, if you say so explicitly in the documentation -that we do not want 100% coverage just to reduce the chance of unnecessary calls to db. 

Also 
"\n" is just line ending should we handle "\r" or "\r\n" ?

Reply:
i think keep alive comment is from this OSS repo in the core: 
https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/internal/builtin_plugins/connection/managed_connection_base.py#L212
and its executed when you open a connection https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/internal/builtin_plugins/connection/impl/router.py#L80

Reply:
OK. Thanks. I've forgotten. Explain in the code with a comment that it's from <link-here> that's why we are checking it.

Another option would be not to rely on the comment. Someone may change it. If we could change it to just check for "select 1" (which is a frequently used connectivity check not just in vdk) might be better.

Reply:
I agree for the comment, but wouldn't select 1 pass also queries like:
SELECT 1 FROM table;

Reply:
That depends on the implementation, right?